packages/commonwealth/client/scripts/state/api/discord/removeDiscordBotConfig.ts

```diff
@@ -0,0 +1,31 @@
+import { useMutation } from '@tanstack/react-query';
+import axios from 'axios';
+import app from 'state';
+import { ApiEndpoints, queryClient } from 'state/api/config';
+
+interface RemoveDiscordBotConfigProps {
+  communityId: string;
+}
+
+const removeDiscordBotConfig = async ({
+  communityId,
+}: RemoveDiscordBotConfigProps) => {
+  await axios.post(`${app.serverUrl()}/removeDiscordBotConfig`, {
```

Comment:
lets move `/removeDiscordBotConfig` as const to `/client/scripts/state/api/config.ts`